scraper_service/README.md

```diff
@@ -69,36 +68,38 @@ zip -r my-Python36-Pandas23.zip .
 Once you have created a zipfile with the fake_useragent library, upload it to your Lambda function with the UI.
 
 
-3. Amazon S3
------------------
+#### 3. Amazon S3
 
 We chose Amazon S3 to store scraped images. After setting up an S3 bucket, we needed to grant our scraper service "full access" to the S3 bucket. This enables the service to write to S3. Access to S3 can be granted through Amazon's IAM UI.
 
-Running the Service
-====================
-
-The scraper service does not require any inputs, so it can be triggered at any time. We trigger the service in one of two ways:
-
-
-a. On a Schedule
------------------
+## INPUT
+The Scraper services subscribes to the AWS SNS topic *triggerScrape* which will contain the variable *phone_number* which contains the user's phone number.
 
-AWS Cloudwatch provides a mechanism to schedule the execution of AWS Lambda functions. We scheduled the service to run every 30 minutes.
+Below is a portion of the JSON received by this service:
```

Comment:
I would change this to: "The following JSON message is received by this service:"